sentinelone/plugin.spec.yaml

```diff
@@ -513,6 +513,505 @@ types:
       type: boolean
       description: File malicious content
       required: false
+  agents_update_system_data:
+    fileName:
+      title: File name
+      type: string
+      required: false
+    path:
+      title: Path
+      type: string
+      required: false
+    packageId:
+      title: Package ID
+      type: string
+      required: false
+    osType:
+      title: OS type
+      type: string
+      required: false
+      enum:
+        - windows_legacy
+        - macos
+        - linux
+        - windows
+  agents_firewall_rules_data:
+    format:
+      title: Format
+      description: Desired firewall configuration format. Use native to get configuration file in native format
+      type: string
+      required: false
+      enum:
+        - native
+        - asset
+    state:
+      title: State
+      description: Desired firewall configuration state. Use initial to get the firewall configuration that existed prior to installing the agent (Note - initial requires native format
+      type: string
+      required: false
+      enum:
+        - initial
+        - current
+  agents_fetch_applications_filter:
+    isDecommissioned:
+      title: Is decomissioned
+      description: Include active decommissioned or both
+      type: '[]boolean'
+      required: false
+    isUninstalled:
+      title: Is uninstalled
+      description: Only include uninstalled agents
+      type: boolean
+      required: false
+    query:
+      title: Query
+      description: A free-text search term, will match applicable attributes (sub-string match). Note - Devices physical addresses will be matched if they start with the search term only (instead of being matched if they contain the term)
```

Comment:
```suggestion
      description: A free-text search term, will match applicable attributes (sub-string match). Note - devices physical addresses will be matched if they start with the search term only (instead of being matched if they contain the term)
```